### PR TITLE
Requirements update for astrodbkit2

### PR DESCRIPTION
**Pull request by dr-rodriguez**
We want to make sure we are using the latest stable version of astrodbkit2 from pypi rather than using the version from git. This allows us to make modifications to astrodbkit2 without breaking the website.
I've left it unpinned in terms of a version since I think getting the latest stable version should be good enough, but if we make breaking changes we'll want to pin it at that time.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 asdf==2.7.3
-git+https://github.com/dr-rodriguez/AstrodbKit2.git@main#egg=astrodbkit2
+astrodbkit2
```
Comment (Will-Cooper) [inline requirements.txt:2]: I would suggest requiring a minimum version so `astrodbkit2>=X.X.X`

**Review by Will-Cooper — COMMENTED** [review]
While we're here in requirements.txt, we could roll in #109 and #112?

**Comment by dr-rodriguez** [comment]
Tried to update to 3.10 (I haven't tested astrodbkit2 with 3.11 yet) and have the latest bokeh and specutils, but there are tons of dependency changes needed. I did get it to partially work, but ran into bigger issues: your construction of simple_root and linking is not stable enough. Running in a linux container from Windows does not work since links are different in that file system. We should rework this implementation since it's not clear to me why you need a link back to itself to make this work.
Bokeh 3 also changed some of the ways to produce hover tooltips so even after removing simple_root I had issues loading the application.

**Comment by Will-Cooper** [comment]
We also need to patch a security exploit by upgrading to `pillow>=9.2.0`.

**Comment by Will-Cooper** [comment]
I'll try and find the time in the next few days to have a play with dependencies and simple_root

**Review by Will-Cooper — APPROVED** [review]
lgtm

**Pull request merged**